JavaBytecodeCompiler/src/main/java/org/openzen/zenscript/javabytecode/compiler/JavaMethodBytecodeCompiler.java

```diff
@@ -405,6 +405,14 @@ public Void builtinStaticMethod(BuiltinMethodSymbol method, CallArguments args)
 				javaWriter.invokeVirtual(STRING_SUBSTRING);
 				return null;
 			}
+			case STRING_TO_UPPER_CASE:
+				arguments[0].accept(expressionVisitor);
+				javaWriter.invokeVirtual(STRING_TO_UPPER_CASE);
+				return null;
+			case STRING_TO_LOWER_CASE:
+				arguments[0].accept(expressionVisitor);
+				javaWriter.invokeVirtual(STRING_TO_LOWER_CASE);
+				return null;
```

Comment:
This should be added to org.openzen.zenscript.javashared.JavaBuiltinModule#generate instead.